featuretools/primitives/standard/transform_primitive.py

```diff
@@ -298,8 +509,17 @@ def get_function(self):
 
 
 class Percentile(TransformPrimitive):
-    """For each value of the base feature, determines the percentile in
-        relation to the rest of the feature.
+    """Determines the percentile rank for each value of the base feature.
```

Comment:
rather than say base feature, we should say "Determines the percentile rank for each value in a list".

the usage of base feature is old and should be replaced everywhere in the file

Reply:
Am I correct in assuming you only want this replaced in the user-facing documentation and not also in the code that uses the base_feature terminology?

Reply:
correct. let's just update the docstrings.